_data/menu.yml

```diff
@@ -11,3 +11,4 @@ items:
   - extending-nixer.md
   - additional-features.md
   - resources.md
+  - nixer-properties.md
```

Comment:
Great content. I would put it under "Extending Nixer". Maybe we need to change name of the section "Extending Nixer" to something else. But I don't this this section should exist on the top level menu.

Reply:
Moved under Extending Nixer.